### DISCUSSION TITLE
SmartMeter SML: Send MQTT message on new SML message

### DISCUSSION THREAD

**Discussion by renne** [arendst/Tasmota#17428] [Q&A] [answered]
Currently the minimum TelePeriod value is 10 seconds.
This is enough for tracking energy consumption but does not suffice for tracking current power consumption.

Please add an option to send a MQTT message whenever a new push telegram is received from the energy meter.

**Comment by gemu2015** [comment, marked as answer]
This is implemented already from the beginning, see docs
Just add 16 to the decimal resolution
May be the docs should be clearer about this functionality
You may improve docs by editing in github
